bunsen-avro/src/main/java/com/cerner/bunsen/avro/tools/GenerateSchemas.java

```diff
@@ -77,4 +78,19 @@ public static int main(String[] args) {
 
     return 0;
   }
+
+  /**
+   * Helper function to extract contained resources from resource string.
+   * 
+   * @param key the string containing resource url(s)
+   * @return the list of contained urls
+   */
+  private static List<String> generateContainedUrls(String key) {
```

Comment:
@RamIyer1998 Is this method has any junit tests.

Reply:
It does not, I'll add them now!

Reply:
93f1cc8